### PR TITLE
Fix some of the issues with tooltips going crazy

### PR DESCRIPTION
**Pull request by DuncanSmith1126**
Also update the font size

### CODE REVIEW

File: pkg/sloop/webfiles/sloop_ui.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
  */
 
 const displayMaxX = document.documentElement.clientWidth;
+const displayMaxY = document.documentElement.clientHeight;
```
Comment (duke-harlan) [inline pkg/sloop/webfiles/sloop_ui.js]: do these need to be reset if the window is resized?

File: pkg/sloop/webfiles/sloop_ui.js [hunk 2]
```diff
@@ -32,25 +33,22 @@ let cmpFn = noSortFn;
...
+    let data = processAndSortResources(result);
+    let svg = render(data);
     bindMouseEvents(svg);
     appendAxes(svg);
     renderTooltip();
 });
 
-function render(result) {
-    let data = processAndSortResources(result);
+function render(data) {
```
Comment (duke-harlan) [inline pkg/sloop/webfiles/sloop_ui.js]: no objection - wondering about the **why** of pulling `processAndSortResources` out, I'm not seeing any other change that leverages this

File: pkg/sloop/webfiles/sloop_ui.js [hunk 3]
```diff
@@ -257,6 +260,7 @@ function bindMouseEvents(svg) {
                     detailedToolTipIsVisible = true;
                     tooltip.html(result);
                     evalJSFromHtml(result);
+                    positionTooltip(tooltipX, tooltipY)
```
Comment (duke-harlan) [inline pkg/sloop/webfiles/sloop_ui.js]: is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?

File: pkg/sloop/webfiles/sloop_ui.js [hunk 4]
```diff
@@ -242,9 +240,14 @@ function bindMouseEvents(svg) {
                 }
             );
             tooltip.html(resourceBarHtml);
+            positionTooltip(tooltipX, tooltipY);
```
Comment (duke-harlan) [inline pkg/sloop/webfiles/sloop_ui.js]: is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?

File: pkg/sloop/webfiles/sloop_ui.js (hunk 3 above)
Comment (duke-harlan) [reply pkg/sloop/webfiles/sloop_ui.js]: not sure I understand the need for the `else` half of this if

File: pkg/sloop/webfiles/sloop_ui.js [hunk 5]
```diff
@@ -429,5 +401,65 @@ function createTooltip(el) {
...
+        let resourceBarHtml = getResourceBarContent(
+            {
+                title: d.text,
+                kind: d.kind,
+                namespace: d.namespace,
+                time: theTime
+            }
+        );
+        tooltip.html(resourceBarHtml);
+        positionTooltip(tooltipX, tooltipY);
```
Comment (duke-harlan) [inline pkg/sloop/webfiles/sloop_ui.js]: is this going to cause a flicker (visible for a short time in original location/size, then move/resize based on additional data)?

File: pkg/sloop/webfiles/sloop_ui.js (hunk 2 above)
Comment (DuncanSmith1126) [reply pkg/sloop/webfiles/sloop_ui.js]: Mostly clarity and future use -esp for animations.

File: pkg/sloop/webfiles/sloop_ui.js [hunk 6]
```diff
@@ -406,7 +374,8 @@ function createResourceBar(d) {
         .attr("x", isLabelRight ? sx - 5 : sx + w + 5)
         .attr("fill", "black")
         .style("text-anchor", isLabelRight ? "end" : "start")
-        .style("dominant-baseline", "hanging");
+        .style("dominant-baseline", "hanging")
+        .style("font-size", "14");
```
Comment (thomashargrove) [inline pkg/sloop/webfiles/sloop_ui.js]: Can we use CSS for the font sizes?  If not, can we define the value at the top of this file so its easy to find and tweak?

**Comment by thomashargrove** [comment]
This is a great improvement for viewing resource details on the main page, but now when I click "Open in new Tab" it renders with a strange half-page on the left.  Is there a way to avoid that?

**Pull request merged**